MARIE_AND_BERT/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.8 as flask
+ENV UWSGI_CHEAPER 1
+ENV UWSGI_PROCESSES 2
+WORKDIR /app
+
+RUN cp -r TheWorldAvatar/MARIE_AND_BERT/Marie ./Marie  \
+&& cp -r TheWorldAvatar/MARIE_AND_BERT/uwsgi.ini ./uwsgi.ini \
+&& cp -r TheWorldAvatar/MARIE_AND_BERT/static ./static \
+&& cp -r TheWorldAvatar/MARIE_AND_BERT/templates ./templates \
+&& cp -r TheWorldAvatar/MARIE_AND_BERT/main.py ./main.py \
+&& cp -r TheWorldAvatar/MARIE_AND_BERT/requirements_linux.txt ./requirements_linux.txt
```

Comment:
Have you tested building the image again (since removing the git clone command)? Just from looking at them, I very much doubt that these instructions are going to work without modification... You need to copy the right ( ! ) folders/files from the local file system into the right place within the image.